### PR TITLE
Version 2.0 merge

### PR DESCRIPTION
**Pull request by yurydelendik**

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -1047,17 +1035,11 @@ let PDFViewerApplication = {
...
           return;
         }
-        javaScript.some((js) => {
-          if (!js) { // Don't warn/fallback for empty JavaScript actions.
-            return false;
-          }
-          console.warn('Warning: JavaScript is not supported');
-          this.fallback(UNSUPPORTED_FEATURES.javaScript);
-          return true;
-        });
```
Comment (Snuffleupagus) [inline web/app.js]: The `javaScript.some((js) => { ... });` changes shouldn't be reverted (note PR #9009).

**Comment by Snuffleupagus** [comment]
I know that we treat https://github.com/mozilla/pdf.js/blob/985c700bd564cc0449b3932b3f4d51801104360e/package.json#L3 more or less like a dummy field, but nonetheless, do we want to update that as well here?

**Comment by yurydelendik** [comment]
Cross-link to https://github.com/mozilla/pdf.js/projects/5

**Pull request merged**